modules/vstudio/vs2010.lua

```diff
@@ -85,16 +85,18 @@
 			end
 		end
 
-		-- Skip generation of empty packages.config files
-		local packages = p.capture(function() vstudio.nuget2010.generatePackagesConfig(prj) end)
-		if #packages > 0 then
-			p.generate(prj, "packages.config", function() p.outln(packages) end)
-		end
+		if _ACTION < "vs2017" then
```

Comment:
PackageReference is not yet supported for C++ projects, so this will have to be changed to check for that:

```lua
if _ACTION < "vs2017" or p.project.iscpp(prj) then
```